### ISSUE TITLE
Edit won't show fields until i change the depends on field

### ISSUE THREAD

**Issue by sirikkoster** [epartment/nova-dependency-container#88]
If i create an record everything goes fine,
but when i edit a record the additional field is not visible

The `use HasDependencies;` is available in this resource

```php
MorphTo::make('Data', 'collectable')
    ->types([
        Movie::class,
        TvShow::class
    ])->searchable(),
    NovaDependencyContainer::make([
        Text::make('Name')
    ])
    ->dependsOn('collectable', 'Movie')
```
Am i doing something wrong or is this an issue

**Comment by wize-wiz** [comment]
@sirikkoster I was able to reproduce the problem.

**Comment by wize-wiz** [comment]
@sirikkoster Problem is solved but I just need you to verify one thing. If the dependency is met (as in satisfied), do you see this field appear on the detail page? I currently don't.

**Comment by sirikkoster** [comment]
currently i do not see it on the detail page

**Comment by wize-wiz** [comment]
@sirikkoster Should be fixed in [1.2.8](https://github.com/epartment/nova-dependency-container/releases/tag/1.2.8), please confirm and if so, close this issue.

**Comment by sirikkoster** [comment]
@wize-wiz fixed
